src/dawn/CodeGen/CXXNaive/ASTStencilFnParamVisitor.cpp

```diff
@@ -0,0 +1,58 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
alright, that is fine. Before it was ASTCXXNaiveStencilFunctionParamVisitor... a bit longy, but since I removed the CXXNaive we have some free chars ;)